test/functional/import-rescan.py

```diff
@@ -179,6 +179,8 @@ def run_test(self):
             else:
                 variant.check()
 
+        assert_equal(try_rpc(self.nodes[0].abortrescan)[1], None)
```

Comment:
I believe there is no need to `try_rpc` since `abortrescan` returns bool. For that reason, add 2 tests, one for each return value.